client/src/components/Button/FeedbackButton.js

```diff
@@ -16,8 +16,8 @@ const FeedbackButton = styled(Button)`
     background-color: ${royalBlue};
     color: ${white};
   }
-  &.am-button-ghost:before {
-    border: none;
+  &.am-button {
+    border: solid ${royalBlue} !important;
```

Comment:
@robinv85 any thoughts of using `!important`? I have come across this before, and was strongly suggested not to use it and look for alternatives... 

Also @Joeu I think the design doesn't look good. 
So here it is how it is right now but just with the borders being wrong. 
![image](https://user-images.githubusercontent.com/25408660/84058887-d1936b80-a987-11ea-9e17-e0e585450b67.png)

Here it is how it looks with your new change -> 
![image](https://user-images.githubusercontent.com/25408660/84059004-f12a9400-a987-11ea-9aa2-3d5ce31f65e2.png)

@galeza @Yu-Alexa-Zhao I think this should look like it was but with the corners not being cut off.. Not sure though

Reply:
Hi @idjevm @Joeu 

This is from Figma design:
![Screen Shot 2020-06-08 at 19 02 42](https://user-images.githubusercontent.com/21987083/84059408-da516500-a9ba-11ea-962a-ffb927e59d65.png)

Reply:
Cool! That means that now it looks the right way. 

@Joeu then for now just remove the mq import and wait to see what @robinv85  has to say on `!important` tag...

Reply:
Thanks, @idjevm . I was testing over the media queries approach and forgot to remove it. Just pushed the change.
About the **!important**, the thing is that without it, the property will be set by antd lib.

Reply:
Yes I believe the Figma & Zeplin screenshot you have is the correct design

Reply:
Yes I believe the figma/zeplin screenshot is the correct one

Reply:
This rule is only overwriting the rule from line 8. the important can be deleted. And it's only overwriting the border width that can be changed in the same line 8. All this line can be deleted.

Reply:
@silenceway , applying the changes you suggested and using the _Toggle device toolbar_ on Chrome, the issue still remains. I also tested with my mobile.

That's why I needed to use the !important, even knowing that normally it should be avoided.

![Screenshot from 2020-06-09 00-46-38](https://user-images.githubusercontent.com/1004896/84104073-22887b80-a9eb-11ea-9a07-e6230158f93b.png)

Reply:
If it's happening only on mobile, you can check the rule and try to override without an important. This works:
`  html body &.am-button {
    border: 0.3rem solid ${royalBlue};
  }`

Reply:
Yeah using `!important` is generally not recommended because it can sometimes cause more problems than solve them.
For this particular example it's not really a problem because it's overriding styles from Ant Design.
It's better to learn about the[ CSS specificity ](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)to understand why using `!important` is not recommended, but that's especially for reusable components, here it's only for the end result on the home page so probably nothing else will extend those styles.

Reply:
> If it's happening only on mobile, you can check the rule and try to override without an important. This works:
> ` html body &.am-button { border: 0.3rem solid ${royalBlue}; }`
> And I think the other rule can be removed. (Rule in line 8).

Thanks for this useful tip, @silenceway . Works like a charm and avoided overriding the specificity.